runner/src/main/kotlin/com/cosmotech/runner/service/RunnerService.kt

```diff
@@ -298,7 +298,7 @@ class RunnerService(
                 }
         val solution =
             workspace?.solution?.solutionId?.let {
-              solutionApiService.findSolutionById(organization?.id!!, it)
+              solutionApiService.getSolution(organization?.id!!, it)
```

Comment:
This is more linked to other PRs but:
- linked to previous one on Organization: `.id!!` should `.id` (here and just below)
- linked to the future one on Runner: the `organization` member can maybe be made non optional? Not sure how a `RunnerService` without organization could work, but I may be wrong. This can probably be checked/done in the dedicated Runner refacto.